lib/basedriver/driver.js

```diff
@@ -140,6 +171,14 @@ class BaseDriver extends MobileJsonWireProtocol {
   // and ensuring that we execute commands one at a time. This method is called
   // by MJSONWP's express router.
   async executeCommand (cmd, ...args) {
+    // get start time for this command, and log in special cases
+    let startTime = Date.now();
```

Comment:
@imurchie i like process.hrtime, but didn't know of its value when what we are generating at the end of the day is not an interval but an absolute time. i can certainly change it to hrtime, but i think ms resolution is plenty good for the kinds of timings we're thinking about. willing to be persuaded though.